exporter/zipkin/lib/opentelemetry/exporter/zipkin/transformer.rb

```diff
@@ -0,0 +1,180 @@
+            # https://github.com/open-telemetry/opentelemetry-specification/blob/84b18b23339dcc0b1a9d48f976a1afd287417602/specification/trace/sdk_exporters/zipkin.md#status
+            # https://github.com/openzipkin/zipkin-ruby/blob/7bedb4dd162c4cbeffc7b97dd06c8dbccbfbab62/lib/zipkin-tracer/trace.rb#L259
+            # https://github.com/open-telemetry/opentelemetry-collector/blob/81c7cc53b7067fbf3db4e1671b13bbe2796eb56e/translator/trace/zipkin/traces_to_zipkinv2.go#L144
+            tags[ERROR_TAG_KEY] = if span_data.status.description.nil? || span_data.status.description.empty?
+                                    ''
+                                  else
+                                    span_data.status.description
+                                  end
+            tags[STATUS_CODE_NAME] = STATUS_ERROR
+          elsif span_data.status&.code && span_data.status&.code == OpenTelemetry::Trace::Status::OK
+            tags[STATUS_CODE_NAME] = STATUS_OK
+          end
+        end
+
+        def add_conditional_tags(zipkin_span, span_data, tags, service_name)
+          zipkin_span['tags'] = tags unless tags.empty?
+          zipkin_span['kind'] = KIND_MAP[span_data[:kind]] unless span_data[:kind].nil?
+          zipkin_span['parentId'] = span_data.parent_span_id.unpack1('H*') unless span_data.parent_span_id.nil?
+          zipkin_span['localEndpoint'] = endpoint_from_tags(tags, (span_data.attributes && span_data.attributes[SERVICE_NAME_ATTRIBUTE_KEY]) || service_name)
+          # remote endpoint logic https://github.com/open-telemetry/opentelemetry-collector/blob/347cfa9ab21d47240128c58c9bafcc0014bc729d/translator/trace/zipkin/traces_to_zipkinv2.go#L284
+          zipkin_span['remoteEndpoint'] = endpoint_from_tags(tags, nil)
+        end
+
+        def add_annotations(zipkin_span, span_data)
+          # Tried to follow the below
+          # https://github.com/open-telemetry/opentelemetry-collector/blob/347cfa9ab21d47240128c58c9bafcc0014bc729d/translator/trace/zipkin/traces_to_zipkinv2.go#L172
+          # https://github.com/open-telemetry/opentelemetry-collector/blob/347cfa9ab21d47240128c58c9bafcc0014bc729d/translator/trace/zipkin/traces_to_zipkinv2.go#L183-L191
+          # https://github.com/open-telemetry/opentelemetry-specification/blob/cb16422a61219d4fd99b466a70e47cb8af9e26b1/specification/trace/sdk_exporters/zipkin.md#events
+          return if span_data.events.nil? || span_data.events&.size.to_i.zero?
+
+          events = span_data.events&.map do |event|
+            if event.attributes.keys.length.zero?
+              {
+                timestamp: (event.timestamp.to_f * 1_000_000).to_s,
+                value: event.name
+              }
+            else
+              {
+                timestamp: (event.timestamp.to_f * 1_000_000).to_s,
+                value: { "#{event.name}": event.attributes&.transform_values { |val| val.to_s } }.to_json
```

Comment:
```suggestion
                value: { event.name => event.attributes.transform_values(&:to_s) }.to_json
```

AFAICT you don't need to transform the event name to a Symbol.